src/olympia/search/migrations/0001_remove-es-use-classic-similarity-waffle-switch.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.12 on 2020-05-05 13:39
+
+from django.db import migrations
+
+
+def remove_waffle_switch(apps, schema_editor):
```

Comment:
I don't see any model changes associated with this. Just curious, how did Django know to generate this migration?

Reply:
I generated it myself with `makemigrations --empty` and then customized it. It's not super common but that's the way to go for data migrations like this